tests/plugins/test_custom_plugin.py

```diff
@@ -39,3 +44,54 @@ def test_sync_batchnorm_set(tmpdir):
     )
     trainer.fit(model)
     assert plugin.sync_batchnorm is True
+
+
+@RunIf(skip_windows=True, min_torch="1.7.0")
+def test_ddp_fp16_compress_comm_hook(tmpdir):
```

Comment:
these could go into a separate test file for ddp comm hooks

Reply:
nit2: will you add some asserts to these tests? for example, one could test if the correct hooks got registered after trainer.fit()
and probably it's better if we put `fast_dev_run=True`

Reply:
also please note these test won't run in CI because it has torch 1.6, so we will need to run it locally :)

Reply:
yeah, will add assert, btw, found out that these hooks might only work with gpu.